app/views/code_reviews/index.html.erb

```diff
@@ -1,19 +1,17 @@
 <h1>
   Code Reviews
+  <% if can? :manage, CodeReview %>
+    <%= link_to '+ New Code Review', new_code_review_path, class: 'text-success new-link' %>
+  <% end %>
```

Comment:
It's in the h1 because I have the link placed next to the header. I can move it elsewhere if you prefer. See image below.

![screen shot 2015-09-18 at 11 28 30 am](https://cloud.githubusercontent.com/assets/6216666/9967612/6e6057e2-5df8-11e5-863f-544a002d7d71.png)